src/components/ToDoList/TodoWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import uuid from 'uuidv4';
+import { TaskType, TodoWrapperType } from './types';
+import TodoForm from './TodoForm';
+import Todo from './Todo';
+
+function TodoWrapper(props: TodoWrapperType): JSX.Element {
+    const { todoTasks } = props;
+    const [tasks, setTasks] = useState<Array<TaskType>>(todoTasks);
+
+    const addTask = (task: string) => {
```

Comment:
For now idk how to create hooks, because I will need to pass tasks and setTasks functions in each of them, I assume it's necessary to move them in separate file because we will need to use redux/query later on, but for now I have no idea how can I do it without passing tasks and setTasks as a parameters. Could you give me a clue?

Reply:
https://react.dev/learn/reusing-logic-with-custom-hooks 

create new folder hooks in src. Create new file for custom hook. Create custom hook and move react hooks to custom hook.

In this file you'll get const [tasks, createTask, deleteTask, updateTaks] = yourCustomHook();